tests/std/test.lst

```diff
@@ -158,6 +158,7 @@ tests\Dev11_1180290_filesystem_error_code
 tests\GH_000457_system_error_message
 tests\GH_000545_include_compare
 tests\GH_000690_overaligned_function
+tests\LWG3018_shared_ptr_function

```

Comment:
Naming question: Should this be `LWG3018`, `LWG_3018`, or `GH_000539`?

Reply:
I like the current naming. But you're a maintainer so you should decide.

Reply:
The absence of an underscore is consistent with the wg21 link shortener, so let’s go with what you like 😸